We appear to have some build failures.  I haven't looked closely to see if they are potentially caused by me merging main into this PR.  It builds for me on my mac, but it looks like all the Linux builds in CI are unhappy.  Maybe missing headers.

```
FAILED: packager/hls/CMakeFiles/hls_builder.dir/base/master_playlist.cc.o 
/usr/bin/c++ -DCARES_STATICLIB -DCURL_STATICLIB -DLIBXML_STATIC -DSHAKA_IMPLEMENTATION -DSHARED_LIBRARY_BUILD -I/home/runner/work/shaka-packager/shaka-packager/packager/.. -I/home/runner/work/shaka-packager/shaka-packager/packager/../include -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/abseil-cpp/source -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/curl/source/include -I/home/runner/work/shaka-packager/shaka-packager/build/packager/third_party/c-ares/source -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/c-ares/source -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/c-ares/source/include -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/mbedtls/source/include -I/home/runner/work/shaka-packager/shaka-packager/build -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/protobuf/source/src -I/home/runner/work/shaka-packager/shaka-packager/build/packager/third_party/libxml2/source -I/home/runner/work/shaka-packager/shaka-packager/packager/third_party/libxml2/source/include -std=c++11  -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Wall -Wextra -Wnarrowing -Wno-deprecated -Werror -g -std=gnu++17 -fPIC -Wall -Wextra -Werror -Wno-unknown-warning-option -Wno-shorten-64-to-32 -Wno-unused-parameter -Wno-stringop-overflow -MD -MT packager/hls/CMakeFiles/hls_builder.dir/base/master_playlist.cc.o -MF packager/hls/CMakeFiles/hls_builder.dir/base/master_playlist.cc.o.d -o packager/hls/CMakeFiles/hls_builder.dir/base/master_playlist.cc.o -c /home/runner/work/shaka-packager/shaka-packager/packager/hls/base/master_playlist.cc
In file included from /home/runner/work/shaka-packager/shaka-packager/packager/hls/base/master_playlist.cc:7:
/home/runner/work/shaka-packager/shaka-packager/packager/../packager/hls/base/master_playlist.h:39:29: error: ‘vector’ in namespace ‘std’ does not name a template type
   39 |                  const std::vector<CeaCaption>& closed_captions,
      |                             ^~~~~~
/home/runner/work/shaka-packager/shaka-packager/packager/../packager/hls/base/master_playlist.h:12:1: note: ‘std::vector’ is defined in header ‘<vector>’; did you forget to ‘#include <vector>’?
```

There may be other locations as well, so check the CI logs.

--- a/include/packager/hls_params.h
+++ b/include/packager/hls_params.h
@@ -10,6 +10,7 @@
 #include <cstdint>
 #include <optional>
 #include <string>
+#include <vector>
 
 namespace shaka {
 

--- a/packager/app/packager_main.cc
+++ b/packager/app/packager_main.cc
@@ -6,6 +6,7 @@
 
 #include <iostream>
 #include <optional>
+#include <vector>
 
 #if defined(OS_WIN)
 #include <codecvt>

--- a/packager/hls/base/master_playlist.cc
+++ b/packager/hls/base/master_playlist.cc
@@ -9,6 +9,8 @@
 #include <algorithm>  // std::max
 #include <cstdint>
 #include <filesystem>
+#include <set>
+#include <vector>
 
 #include <absl/log/check.h>
 #include <absl/log/log.h>

--- a/packager/hls/base/master_playlist.h
+++ b/packager/hls/base/master_playlist.h
@@ -10,6 +10,7 @@
 #include <filesystem>
 #include <list>
 #include <string>
+#include <vector>
 
 namespace shaka {
 namespace hls {

